src/blocks/blocks/posts/components/layout/index.js

```diff
@@ -176,7 +176,7 @@ export const PostsMeta = ({ attributes, element, post, author, categories }) =>
 		if ( attributes.displayAuthor && undefined !== author ) {
 
 			/* translators: %s Author of the post */
-			postedOn += sprintf( __( ' by %s', 'otter-blocks' ), author.name );
+			postedOn += sprintf( __( 'by %s', 'otter-blocks' ), author.name );
```

Comment:
We use space here to avoid there not being space between this and the previous element.